components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/gen/java/org/wso2/carbon/identity/api/server/userstore/v1/UserstoresApiService.java

```diff
@@ -52,6 +53,8 @@
 
       public Response getUserStoreManagerProperties(String typeId);
 
+      public Response patchAttributeMappings(String userstoreDomainId, List<ClaimAttributeMapping> claimAttributeMapping);
```

Comment:
Don't this needs to be default? Otherwise it'll break the existing implementations right?

Reply:
This is an autogen class by the API generator. Is it required to manually set it as default.